README.md

```diff
@@ -5,107 +5,104 @@
-    enable_amazon_eks_coredns             = true
-    enable_amazon_eks_kube_proxy          = true
-    enable_amazon_eks_aws_ebs_csi_driver  = true
-
-    #K8s Add-ons
-    enable_aws_load_balancer_controller   = true
-    enable_metrics_server                 = true
-    enable_cluster_autoscaler             = true
-    enable_aws_for_fluentbit              = true
-    enable_argocd                         = true
-    enable_ingress_nginx                  = true
-
-    depends_on = [module.eks-blueprints.managed_node_groups]
+module "eks_blueprints_kubernetes_addons" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons"
+
+  eks_cluster_id = module.eks_blueprints.eks_cluster_id
+
+  # EKS Addons
+  enable_amazon_eks_vpc_cni             = true
+  enable_amazon_eks_coredns             = true
+  enable_amazon_eks_kube_proxy          = true
+  enable_amazon_eks_aws_ebs_csi_driver  = true
+
+  #K8s Add-ons
+  enable_argocd                         = true
+  enable_aws_for_fluentbit              = true
+  enable_aws_load_balancer_controller   = true
+  enable_cluster_autoscaler             = true
+  enable_ingress_nginx                  = true
+  enable_metrics_server                 = true
 }
 ```
 
 The code above will provision the following:
 
-✅  A new EKS Cluster with a managed node group.\
-✅  Amazon EKS add-ons `vpc-cni`, `CoreDNS`, `kube-proxy`, and `aws-ebs-csi-driver`.\
-✅  `Cluster Autoscaler` and `Metrics Server` for scaling your workloads.\
-✅  `Fluent Bit` for routing logs.\
-✅  `AWS Load Balancer Controller` for distributing traffic.\
-✅  `Argocd` for declarative GitOps CD for Kubernetes.\
-✅  `Nginx` for managing ingress.
+✅  A new EKS Cluster with a managed node group.  
+✅  Amazon EKS add-ons `vpc-cni`, `CoreDNS`, `kube-proxy`, and `aws-ebs-csi-driver`.  
+✅  `Cluster Autoscaler` and `Metrics Server` for scaling your workloads.  
+✅  `Fluent Bit` for routing logs.  
+✅  `AWS Load Balancer Controller` for distributing traffic.  
+✅  `Argocd` for declarative GitOps CD for Kubernetes.  
+✅  `Nginx` for managing ingress.  
 
 ## Add-ons
-This framework provides out of the box support for a wide range of popular Kubernetes add-ons.
-By default, the [Terraform Helm provider](https://github.com/hashicorp/terraform-provider-helm) is used to deploy add-ons with publicly available [Helm Charts](https://artifacthub.io/).
-The framework provides support for leveraging self-hosted Helm Chart as well.
+
+EKS Blueprints makes it easy to provision a wide range of popular Kubernetes add-ons into an EKS cluster. By default, the [Terraform Helm provider](https://github.com/hashicorp/terraform-provider-helm) is used to deploy add-ons with publicly available [Helm Charts](https://artifacthub.io/).EKS Blueprints provides support for leveraging self-hosted Helm Chart as well.
 
 For complete documentation on deploying add-ons, please visit our [add-on documentation](./docs/add-ons/index.md)
```

Comment:
Lets provide a full link now for Add-on docs 

`https://aws-ia.github.io/terraform-aws-eks-blueprints/add-ons/`

Reply:
just to re-iterate - we should always use fully qualified URL links so that when this is pushed to the registry the links still work (relative links 404 on the Terraform registry)

Reply:
```suggestion
For complete documentation on deploying add-ons, please visit our [add-on documentation](https://aws-ia.github.io/terraform-aws-eks-blueprints/add-ons/)
```